content_sync/utils.py

```diff
@@ -36,3 +36,21 @@ def get_destination_filepath(
         (content.id, content.text_id),
     )
     return None
+
+
+def get_destination_url(
+    content: WebsiteContent, site_config: SiteConfig
+) -> Optional[str]:
+    """
+    Returns the URL a given piece of content is expected to be at
+    """
+    if content.is_page_content:
+        filename = "" if content.filename == "_index" else content.filename
+        dirpath = content.dirpath.replace(content.website.name, "", 1)
```

Comment:
Can we combine this line and the next one, and also check that the search string is a prefix?

    prefix = f'{content.website.name}/{site_config.content_dir}'
    if content.dirpath.startswith(prefix):
        dirpath = content.dirpath.replace(prefix, "", 1)